src/runtime-rs/crates/hypervisor/src/ch/inner_device.rs

```diff
@@ -125,11 +125,13 @@ impl CloudHypervisorInner {
         Ok(())
     }
 
-    pub(crate) async fn get_shared_fs_devices(&mut self) -> Result<Option<Vec<FsConfig>>> {
+    pub(crate) async fn get_network_and_shared_fs_devices(
```

Comment:
Is it possible to use an abstraction for the device type, like returning an enum? Because my concern is if we want to add another device, the function name is going to be like get_network_and_shared_fs_and_XXXX_devices.